cpp/box/Box.h

```diff
@@ -228,33 +228,33 @@ class Box
     /*! \param vecs Vectors of fractional coordinates between 0 and 1 within
      *         parallelepipedal box
      *  \param Nvecs Number of vectors
+     *  \param out The array in which to place the wrapped vectors.
      */
-    void makeAbsolute(vec3<float>* vecs, unsigned int Nvecs) const
+    void makeAbsolute(const vec3<float>* vecs, unsigned int Nvecs, vec3<float>* out) const
     {
         util::forLoopWrapper(0, Nvecs, [=](size_t begin, size_t end) {
             for (size_t i = begin; i < end; ++i)
             {
-                vecs[i] = makeAbsolute(vecs[i]);
+                out[i] = makeAbsolute(vecs[i]);
             }
         });
     }
 
-    //! Compute the position of the particle in box relative coordinates
-    /*! \param p point
-     *  \returns alpha
+    //! Convert a point's coordinate from absolute to relative box coordinates.
```

Comment:
I don't think "relative" and "fractional" mean the same thing. "Relative" suggests a translation (in both physics and CSS contexts), whereas fractional conveys the desired meaning to me.
```suggestion
    //! Convert a point's coordinate from absolute coordinates to fractional box coordinates.
```